server/sonar-web/src/main/js/app/components/help/GlobalHelp.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+// @flow
+import React from 'react';
+import Modal from 'react-modal';
+import classNames from 'classnames';
+import LinksHelp from './LinksHelp';
+import ShortcutsHelp from './ShortcutsHelp';
+import TutorialsHelp from './TutorialsHelp';
+import { translate } from '../../../helpers/l10n';
+
+type Props = {
+  onClose: () => void
+};
+
+type State = {
+  section: string
+};
+
+export default class GlobalHelp extends React.PureComponent {
+  props: Props;
+  state: State = { section: 'shortcuts' };
+
+  handleCloseClick = (event: Event) => {
+    event.preventDefault();
+    this.props.onClose();
+  };
+
+  handleSectionClick = (event: Event & { currentTarget: HTMLElement }) => {
+    event.preventDefault();
+    const { section } = event.currentTarget.dataset;
```

Comment:
Are you sure `dataset` is supported in IE ? If not does babel use a polyfill for it ?